app/models/announcement.rb

```diff
@@ -1,4 +1,7 @@
 class Announcement < ApplicationRecord
+  extend Mobility
+  translates :name
+  translates :description
```

Comment:
An alternative is to set the type explicitly: `translates :description, type: :text`

Not sure if we care about splitting these. Maybe this is to help w querying speed later?

Reply:
I'm seeing [this note](https://github.com/shioyama/mobility#getting-started) in the docs:
> Note: When using the KeyValue backend, use the options hash to pass each attribute's type:
> This is important because this is how Mobility knows to which of the two translation tables it should save your translation.

Although i'm a bit surprised Mobility doesn't just infer this from the column type 🤔 .

Reply:
right. but if we tell it the default is :string, it seems to work that we only need to explicitly state :text if we do NOT want to store it in the string_translations table.

@exbinary this comment was trying to get at asking -- why would we ever want to store something in the text_translations table? my thinking was maybe to have longer strings in a table together so the string table is as fast as it can be? using both string_translations and text_translations doubles the joins, so i wasn't clear on why we would ever choose anything other than the default (string_translations)

Reply:
Oh, i see what you were asking. Sorry, i misunderstood 🤦.

That's a great question.

AFAIC see, the only difference between the two tables is the type of the `value` column, string vs. text.

From https://www.postgresql.org/docs/9.1/datatype-character.html
> Tip: There is no performance difference among these three types, apart from increased storage space when using the blank-padded type, and a few extra CPU cycles to check the length when storing into a length-constrained column. While character(n) has performance advantages in some other database systems, there is no such advantage in PostgreSQL; in fact character(n) is usually the slowest of the three because of its additional storage costs. In most situations text or character varying should be used instead.

Makes me think text is superior and there's no reason to use the string table?